node/clients/index.ts

```diff
@@ -13,16 +13,23 @@ import StorefrontPermissions from './storefrontPermissions'
 import VtexId from './vtexId'
 
 export const getTokenToHeader = (ctx: IOContext) => {
-  const token =
-    ctx.storeUserAuthToken ?? ctx.adminUserAuthToken ?? ctx.authToken
+  const adminToken = ctx.adminUserAuthToken ?? ctx.authToken
```

Comment:
I wonder if we should set this to null in case both adminToken and authToken are null, and only set `VtexIdclientAutCookie` if this is not null.